build-system/tasks/visual-diff/percy-assets-loader.js

```diff
@@ -0,0 +1,67 @@
+const fs = require('fs');
+const glob = require('glob');
+const mime = require('mime-types');
+const path = require('path');
+const walk = require('walk');
+
+const MAX_FILE_SIZE_BYTES = 15728640;
```

Comment:
Where does this limit come from? If it's documented somewhere, add a comment with a link.